WordPress/gutenberg
# Global Styles: Add back the global styles logic that forces the solid border when color or width applied.

Fixes a regression introduced int https://github.com/WordPress/gutenberg/pull/49428
See https://github.com/WordPress/gutenberg/pull/49428#issuecomment-1541441515

## What?

In Global Styles, we have some logic in place that forces the border style to be "solid" when a color/width is applied. This PR just restores that logic that was removed inadvertently. 

## Testing Instructions

1- Open the "button" global styles panel
2- Type 1px in the "border width" input
3- Verify that the "solid" style is applied on the button
